From 80045c41f145e58b4d021e13b08038ad46c3631c Mon Sep 17 00:00:00 2001
From: Daniel Baumann <daniel.baumann@progress-technologies.net>
Date: Wed, 15 Dec 2010 18:37:23 +0100
Subject: [PATCH] Correcting include argument when calling debootstrap in debian template (Closes: #607275).

---
 templates/lxc-debian.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/templates/lxc-debian.in b/templates/lxc-debian.in
index d597321..0b7d711 100644
--- a/templates/lxc-debian.in
+++ b/templates/lxc-debian.in
@@ -121,7 +121,7 @@ openssh-server
     # download a mini debian into a cache
     echo "Downloading debian minimal ..."
     debootstrap --verbose --variant=minbase --arch=$arch \
-	--include $packages \
+	--include=$packages \
 	"$SUITE" "$cache/partial-$SUITE-$arch" $MIRROR
     if [ $? -ne 0 ]; then
 	echo "Failed to download the rootfs, aborting."
-- 
1.7.2.3

