From 0da4eb0952ac86be9f0bf82e1131e5c90d6743d6 Mon Sep 17 00:00:00 2001
From: Daniel Baumann <daniel.baumann@progress-technologies.net>
Date: Thu, 16 Dec 2010 16:07:12 +0100
Subject: [PATCH] Correcting charset argument when calling locale-gen in debian template (Closes: #607273).

---
 templates/lxc-debian.in |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/templates/lxc-debian.in b/templates/lxc-debian.in
index 0b7d711..b3d1634 100644
--- a/templates/lxc-debian.in
+++ b/templates/lxc-debian.in
@@ -76,10 +76,10 @@ EOF
 
     # reconfigure some services
     if [ -z "$LANG" ]; then
-	chroot $rootfs locale-gen en_US.UTF-8
+	chroot $rootfs locale-gen en_US.UTF-8 UTF-8
 	chroot $rootfs update-locale LANG=en_US.UTF-8
     else
-	chroot $rootfs locale-gen $LANG
+	chroot $rootfs locale-gen $LANG $(echo $LANG | cut -d. -f2)
 	chroot $rootfs update-locale LANG=$LANG
     fi
 
-- 
1.7.2.3

