Author: Guido Trotter <ultrotter@debian.org>
Description: Fix too deep lib dir.

diff -Naurp lxc.orig/configure lxc/configure
--- lxc.orig/configure	2011-07-12 16:20:56.361291375 +0200
+++ lxc/configure	2011-07-12 16:52:45.382757699 +0200
@@ -4719,7 +4719,7 @@ fi
 
 
     EXP_VAR=LXCINITDIR
-    FROM_VAR=$libdir/lxc
+    FROM_VAR=$libdir
 
         prefix_save=$prefix
     exec_prefix_save=$exec_prefix
@@ -4747,7 +4747,7 @@ fi
 
 
     EXP_VAR=LXCTEMPLATEDIR
-    FROM_VAR=$libdir/lxc/templates
+    FROM_VAR=$libdir/templates
 
         prefix_save=$prefix
     exec_prefix_save=$exec_prefix
diff -Naurp lxc.orig/configure.ac lxc/configure.ac
--- lxc.orig/configure.ac	2011-07-12 16:20:56.361291375 +0200
+++ lxc/configure.ac	2011-07-12 16:52:45.382757699 +0200
@@ -56,8 +56,8 @@ AS_AC_EXPAND(LXC_GENERATE_DATE, "$(date)
 
 AS_AC_EXPAND(LXCPATH, "${with_config_path}")
 AS_AC_EXPAND(LXCROOTFSMOUNT, "${with_rootfs_path}")
-AS_AC_EXPAND(LXCINITDIR, $libdir/lxc)
-AS_AC_EXPAND(LXCTEMPLATEDIR, $libdir/lxc/templates)
+AS_AC_EXPAND(LXCINITDIR, $libdir)
+AS_AC_EXPAND(LXCTEMPLATEDIR, $libdir/templates)
 AH_TEMPLATE([LXCPATH], [lxc configuration repository])
 AH_TEMPLATE([LXCINITDIR], [lxc-init directory location])
 AH_TEMPLATE([LXCROOTFSMOUNT], [lxc default rootfs mount point])
diff -Naurp lxc.orig/src/lxc/Makefile.am lxc/src/lxc/Makefile.am
--- lxc.orig/src/lxc/Makefile.am	2011-07-12 16:20:56.365291402 +0200
+++ lxc/src/lxc/Makefile.am	2011-07-12 16:52:45.382757699 +0200
@@ -14,6 +14,7 @@ pkginclude_HEADERS = \
 		state.h
 
 sodir=$(libdir)
+pkglibdir = $(libdir)
 # use PROGRAMS to avoid complains from automake
 so_PROGRAMS = liblxc.so
 
diff -Naurp lxc.orig/src/lxc/Makefile.in lxc/src/lxc/Makefile.in
--- lxc.orig/src/lxc/Makefile.in	2011-07-12 16:20:56.365291402 +0200
+++ lxc/src/lxc/Makefile.in	2011-07-12 16:52:45.382757699 +0200
@@ -20,7 +20,6 @@
 VPATH = @srcdir@
 pkgdatadir = $(datadir)/@PACKAGE@
 pkgincludedir = $(includedir)/@PACKAGE@
-pkglibdir = $(libdir)/@PACKAGE@
 pkglibexecdir = $(libexecdir)/@PACKAGE@
 am__cd = CDPATH="$${ZSH_VERSION+.}$(PATH_SEPARATOR)" && cd
 install_sh_DATA = $(install_sh) -c -m 644
@@ -198,6 +197,7 @@ HEADERS = $(pkginclude_HEADERS)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
+pkglibdir = $(libdir)
 ACLOCAL = @ACLOCAL@
 AMTAR = @AMTAR@
 AUTOCONF = @AUTOCONF@
