Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Extending architecture static fallback list for powerpc.

diff -Naurp lxc.orig/templates/lxc-debian.in lxc/templates/lxc-debian.in
--- lxc.orig/templates/lxc-debian.in	2011-07-22 17:32:06.329202804 +0200
+++ lxc/templates/lxc-debian.in	2011-07-22 17:34:35.269941357 +0200
@@ -165,11 +165,11 @@ install_debian()
             arch=`/usr/bin/udpkg --print-architecture`
         else
             arch=$(arch)
-            if [ "$arch" = "i686" ]; then
-                arch="i386"
-            elif [ "$arch" = "x86_64" ]; then
-                arch="amd64"
-            fi
+            case $arch in
+                686)           arch="i386" ;;
+                x86_64)        arch="amd64" ;;
+                ppc)           arch="powerpc" ;;
+            esac
          fi
 
 	echo "Checking cache download in $cache/rootfs-$SUITE-$arch ... "
