Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Don't list vendor specific example template.

diff -Naurp lxc/src/lxc/lxc-create.in lxc.orig/src/lxc/lxc-create.in
--- lxc/src/lxc/lxc-create.in	2012-03-09 13:21:01.354510931 +0100
+++ lxc.orig/src/lxc/lxc-create.in	2012-03-09 13:20:39.210396083 +0100
@@ -52,7 +52,7 @@ help() {
     echo
     if [ -z $lxc_template ]; then
         echo "for template-specific help, specify a template, for instance:"
-        echo "lxc-create -t ubuntu -h"
+        echo "lxc-create -t debconf -h"
         exit 0
     fi
     type ${templatedir}/lxc-$lxc_template >/dev/null
