Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Setting default runlevel to 2 instead of 3.

diff -Naurp lxc.orig/templates/lxc-debian.in lxc/templates/lxc-debian.in
--- lxc.orig/templates/lxc-debian.in	2011-08-07 10:50:50.367130017 +0200
+++ lxc/templates/lxc-debian.in	2011-08-07 10:50:23.130994958 +0200
@@ -38,7 +38,7 @@ configure_debian()
 
     # configure the inittab
     cat <<EOF > $rootfs/etc/inittab
-id:3:initdefault:
+id:2:initdefault:
 si::sysinit:/etc/init.d/rcS
 l0:0:wait:/etc/init.d/rc 0
 l1:1:wait:/etc/init.d/rc 1
