Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Only list directories with lxc-ls (Closes: #629409).

diff -Naurp lxc.orig/src/lxc/lxc-ls.in lxc/src/lxc/lxc-ls.in
--- lxc.orig/src/lxc/lxc-ls.in	2011-12-15 21:55:28.595660522 +0100
+++ lxc/src/lxc/lxc-ls.in	2011-12-26 11:58:01.426679411 +0100
@@ -22,7 +22,7 @@ function get_cgroup()
 	mount_point=`echo "$mount_string" |cut -d' ' -f2`;
 }
 
-ls "$@" $lxcpath
+eval echo $(find $lxcpath -type d -and -not -name lost+found -exec ls "$@" {} \;)
 
 active=$(netstat -xa | grep $lxcpath | \
 	sed -e 's#.*'"$lxcpath/"'\(.*\)/command#\1#');
