Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Setting random root password.

diff -Naurp lxc.orig/templates/lxc-debian.in lxc/templates/lxc-debian.in
--- lxc.orig/templates/lxc-debian.in	2011-10-04 14:16:33.401346086 +0200
+++ lxc/templates/lxc-debian.in	2011-10-04 14:33:46.762470549 +0200
@@ -115,8 +115,10 @@ EOF
     chroot $rootfs /usr/sbin/update-rc.d -f module-init-tools remove # S
     chroot $rootfs /usr/sbin/update-rc.d module-init-tools start 10 3 .
 
-    echo "root:root" | chroot $rootfs chpasswd
-    echo "Root password is 'root', please change !"
+    password="$(dd if=/dev/urandom bs=6 count=1 2> /dev/null | base64)"
+
+    echo "root:$password" | chroot $rootfs chpasswd
+    echo "Root password is '$password', please change !"
 
     return 0
 }
