Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Silence type call for debootstrap.

diff -Naurp lxc.orig/templates/lxc-debian.in lxc/templates/lxc-debian.in
--- lxc.orig/templates/lxc-debian.in	2011-10-04 18:55:40.800391965 +0200
+++ lxc/templates/lxc-debian.in	2011-10-04 18:56:08.456528961 +0200
@@ -356,7 +356,7 @@ if [ ! -z "$clean" -a -z "$path" ]; then
     exit 0
 fi
 
-type debootstrap
+type debootstrap >/dev/null 2>&1
 if [ $? -ne 0 ]; then
     echo "'debootstrap' command is missing"
     exit 1
